UserStory1/ex2.py

```diff
@@ -0,0 +1,29 @@
+arr=['9','5','1','2','3','4','0','-1']
+
+class range_functions:
+    def __init__(self,index1, index2, index3):
+        self.index1=index1
+        self.index2=index2
+        self.index3=index3
+
+    def element_at(self,arr):
```

Comment:
Do not update the function signatures. Follow the provided signature.
`element_at()` should take two parameters not one.
Same for other functions.